homework_1/zad1/solution/game.go

```diff
@@ -0,0 +1,28 @@
+package solution
+
+import (
+	"github.com/pkg/errors"
+	"strconv"
+)
+
+func Game(first int, last int ) ([]string, error){
```

Comment:
Similar to the other comment, there shouldn't be a whitespace before `)` and there should be a whitespace before `{` (I believe both should be fixed automatically by `go fmt`).